modules/ROOT/pages/managing-apis.adoc

```diff
@@ -30,10 +30,25 @@ Shows the APIs that exist in your Exchange organization but have not been added
 
 //image::aeh-api-description.png[View API Description]
 
+== Opt In to Increase the Number of APIs In the Portal
+Enable portal scalability to increase the maximum number of APIs available in your portal to 1,000. Administrators opt-in from the *API management* page. 
```

Comment:
```suggestion
Enable portal scalability to increase the maximum number of APIs available in your portal to 1,000. Administrators opt in from the *API management* page. 
```

Reply:
I guess it's really lc (management)

Reply:
Yes, unfortunately the UI text is in sentence case. I’ve pushed for uppercase, but it’s not their priority to change.

Reply:
I think this change (removing the dash) is applicable, though.